build.sbt

```diff
@@ -6,6 +6,9 @@ ThisBuild / startYear := Some(2021)
 ThisBuild / crossScalaVersions := List("3.3.5", "2.12.20", "2.13.16")
 ThisBuild / tlVersionIntroduced := Map("3" -> "1.0.2")
 
+ThisBuild / libraryDependencySchemes +=
+  "org.scala-native" %% "test-interface_native0.5" % VersionScheme.Always
+
```

Comment:
Hi @iRevive ,
Is this necessary because ScalaCheck v1.18.1 depends on Native v0.5.5 ?
If it is the only reason, I think it makes sense to arrange ScalaCheck v1.18.2 that would depend on v0.5.6.
Wdyt?

Perhaps it would help synchonize [discipline-munit](https://github.com/typelevel/discipline-munit) too.